lua/lint.lua

```diff
@@ -150,9 +163,9 @@ function M.try_lint(names, opts)
       notify(handle_or_error --[[@as string]], vim.log.levels.WARN)
     end
   end
-  return {
-    handles = handles
-  }
+
+  buffer_to_running_handles[bufnr] = handles
+
```

Comment:
This would be leaking the handles as they never get cleaned up if a buffer is closed/deleted.